package.json

```diff
@@ -4,6 +4,7 @@
     "format:fix": "prettier . --write"
   },
   "dependencies": {
+    "type-fest": "^2.13.0",
```

Comment:
`type-fest` should be a `devDependency` and we should use `import type` when using it. This is because it doesn't produce any JS code. Also, to keep with the system, it should be installed as a dependency on each of the depending projects, to make it easier to detect unused dependencies in the future, but we'll keep it here for now.